cpp/arcticdb/CMakeLists.txt

```diff
@@ -660,7 +662,7 @@ if(${TEST})
                 )
     endif()
 
-    gtest_discover_tests(test_unit_arcticdb)
+    gtest_discover_tests(test_unit_arcticdb DISCOVERY_TIMEOUT 30)
```

Comment:
@alexowens90 Is this effectively the same change you've made?

Reply:
It isn't needed now the build directory is mounted as tmpfs in the dev container, can remove